framework/src/dm/arch/sidk_s5jt200/s5j_dm_connectivity.c

```diff
@@ -52,6 +52,10 @@
 #define DM_BSSID_LEN            18
 #define DM_NET_DEVNAME          "wl1"
 
+#if defined(CONFIG_DM) && !defined(CONFIG_DM_WIFI)
```

Comment:
As per external/Kconfig, enabling DM_WIFI automatically enables the three configs mentioned above. Therefore, it is not necessary to check all configs.